### PR TITLE
Create core members feature test

### PR DESCRIPTION
**Pull request by susannasouv**

### CODE REVIEW

**Review by LizzySteger — COMMENTED** [review]
Should add in pending step definitions for "I am not an admin" and "the page should not have a input" ?

**Comment by susannasouv** [comment]
@LizzySteger the "I am not an admin" step has been pending since last iteration, and "the page should not have a input" step comes from web_steps. Seems like a good call to define "the page should not have a input" as pending on our own though.

**Review by LizzySteger — APPROVED** [review]
LGTM!

File: features/core_members_admin.feature [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Feature: Core Members admin control
+  As a Core Member of Da-tong
+  So that I can decide which users are and are not core members
+  I should be able to access a page that allows me to edit who the core members are
+
+Scenario: Visiting the Core Members edit page without admin permissions
+  Given I am not a Core Member
+  And I am on the Edit Core Members page
```
Comment (aganpule) [inline features/core_members_admin.feature:8]: Does this mean that non Core-members can still visit the edit page?
Can we just show a message on the edit page to non core members that says "You do not have access to this page"

**Review by aganpule — APPROVED** [review]
looks good

File: features/core_members_admin.feature (hunk 1 above)
Comment (LizzySteger) [reply features/core_members_admin.feature:8]: Good idea! I'm pretty sure the "I am on the Edit Core Members page" will attempt to go to the URL so we can add a step after to make sure that message is seen?

**Pull request merged**